api/spec/v1.yaml

```
@@ -2089,6 +2089,10 @@ components:
             applicable. The meaning varies based on the transaction method.
             Usually in native denomination, ParaTime units. As a string.
           example: "100000001666393459"
+        encryption_envelope:
+          description: |
+            The encrypted transaction envelope.
```

```suggestion
            The encrypted data and result of the transaction. Only present for encrypted transactions in confidential EVM runtimes like Sapphire.
```

@pro-wh this is specific to EVM, right? Cipher would have slightly different fields, and we'd want to store/present them differently?